control-plane/config/sink/100-kafka-sink.yaml

```diff
@@ -0,0 +1,157 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: kafkasinks.eventing.knative.dev
+  labels:
+    duck.knative.dev/addressable: "true"
+    eventing.knative.dev/release: devel
+spec:
+  group: eventing.knative.dev
+  names:
+    kind: KafkaSink
+    plural: kafkasinks
+    singular: kafkasink
+  scope: Namespaced
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      subresources:
+        status: { }
+      schema:
+        openAPIV3Schema:
+          description: 'Kafka Sink is Addressable, it receives events and send them to a Kafka topic.'
+          type: object
+          properties:
+            spec:
+              description: 'Spec defines the desired state of the Kafka Sink.'
+              type: object
+              required:
+                - topic
+                - numPartitions
+                - replicationFactor
+                - bootstrapServers
+              properties:
+                topic:
+                  description: 'Topic name to send events.'
+                  type: string
+                numPartitions:
+                  description: 'Number of topic partitions'
+                  type: integer
+                  format: int32
+                replicationFactor:
+                  description: 'Topic replication factor'
+                  type: integer
+                  format: int32
+                bootstrapServers:
+                  description: 'A comma separated list of host/port pairs to use for establishing the initial connection to the Kafka cluster.'
+                  type: string
+                contentMode:
+                  description: 'CloudEvent content mode of Kafka messages sent to the topic.
```

Comment:
to the `contentMode` schema also add `default: structured` (not only in the description)

Reply:
Is that not enough? https://github.com/knative-sandbox/eventing-kafka-broker/pull/134/files#diff-caaba0d14cbd29a23ec3280c8c8fe4abR69?

Reply:
ah ok, sorry ...